include/endian.h

```diff
@@ -146,12 +146,24 @@
 
 #define betoh16               be16toh
 #define letoh16               le16toh
+#define bemtoh16(x)           htobe16(*(uint16_t *)(x))
```

Comment:
Why this is not depending on big or little endian?

Reply:
Basic macros are already defined

Reply:
Sorry. Missed that